analytics/partner.js

```diff
@@ -8,4 +8,16 @@
   analyticsHooks.on('partner:non-claimed', function (options) {
     analytics.track('Non-claimed partner page', _.extend(options, {nonInteraction: 1}))
   })
+
+  $('#partner-contact .email-gallery').click(function () {
+    analytics.track('Clicked Contact Gallery Via Email', {
+      gallery_id: sd.CURRENT_PATH.split('/')[1] // gallery id
```

Comment:
Is this `CURRENT_PATH` splitting the way ID grabbing is done in all analytics code?